icepyx/tests/test_behind_NSIDC_API_login.py

```diff
@@ -32,8 +34,8 @@ def session(reg):
 
 
 def test_get_custom_options_output(session):
-    obs = is2ref._get_custom_options(session, "ATL06", "005")
-    with open("./icepyx/tests/ATL06v05_options.json") as exp_json:
+    obs = is2ref._get_custom_options(session, "ATL06", "006")
+    with open("./icepyx/tests/ATL06v06_options.json") as exp_json:
```

Comment:
What would be the behavior if a script points to V005?

Reply:
As long as it's still available from NSIDC, the version specified will be ordered. If not, whatever error comes from NSIDC in the request will be surfaced. The default is to the most recent version, so we hard code the version into the test so that when a new version is released any changes in the options don't cause an error before we can manually update the json.